### ISSUE TITLE
Some of the data in the response was lost.

### ISSUE THREAD

**Issue by DarkoreXOR** [TwitchLib/TwitchLib.Api#99]
I make a request through `"api.V5.Games.GetTopGamesAsync(...)"`, part of the data in the class `"TwitchLib.Api.V5.Models.Games.Game"` disappears (I get `"id == 0"` and `"viewers == 0"`).

I checked the response format on the website using the link below and in the TwitchLib.Api code. The response format may have changed (the id was renamed _id, and the viewers were completely removed).

[https://dev.twitch.tv/docs/v5/reference/games/](https://dev.twitch.tv/docs/v5/reference/games/)

Can you confirm the problem?

**Comment by Mahsaap** [comment]
All the parsing look correct in code. But it does seem like the game object is pulling more then what is available on the return object now.

For Reference game.cs doesn't have "viewers", _id is still there.  if remember later ill PR it in Dev branch.

But I am not sure why you would be getting 0 on things.

**Comment by DarkoreXOR** [comment]
Oh. My bad. I downloaded the nuget package version 3.0.1 in which this fix did not exist yet. And when I looked at the code, on github from the main page of TwitchLib I got through the TwitchLib.Api dependency on the commit version 3.0.0 and thought that the problem still exists. With _id figured out.

But I think `viewers`  should be removed from `"TwitchLib.Api.V5.Models.Games.Game"`. `Viewers` is still in `"TwitchLib.Api.V5.Models.Games.TopGame"` and it works correctly.

I have no more questions. Should I close this issue?
